lib/src/components/NxPolicyViolationIndicator/NxPolicyViolationIndicator.scss

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+@import '../../scss-shared/nx-variables';
+@import '../../scss-shared/nx-text-helpers';
+
+.nx-policy-violation-indicator {
+  --policy-indicator-highlight-colour: #{$nx-threat-color-unspecified};
+  align-items: center;
+  border: 1px solid var(--policy-indicator-highlight-colour);
+  border-radius: $nx-border-radius;
+  display: flex;
+  height: 30px;
+  margin: 0 0 $nx-spacing-xs $nx-spacing-md;
+  overflow: hidden;
+  width: max-content;
+}
+
+.nx-policy-violation-indicator__badge {
+  align-self: stretch;
+  align-items: center;
+  background-color: var(--policy-indicator-highlight-colour);
+  display: flex;
+
+  .nx-icon {
+    color: #fff;
+  }
+}
+
+.nx-policy-violation-indicator__text {
+  @include semi-bold();
+  @include nx-truncate-ellipsis;
+
+  font-size: $nx-font-size-s;
+  margin: 0 $nx-spacing-md;
+  text-transform: uppercase;
+  // max width of element from design is 859px, subtract the badge, the padding, and the borders.
+  max-width: 793px;
```

Comment:
I'm thinking about this more and wondering if what we should really do is make the indicators inline-block elements, considering their typical placement, and also give them a max-width of 100%.  What you have now doesn't really address the issue at hand, which is the possibility of the indicator being wider than its container.  For example (using a hardcoded 400px width on the `.gallery-example-live`):
![image](https://user-images.githubusercontent.com/3630091/100762090-96677780-33c1-11eb-8107-9eddcf7272c9.png)

Reply:
> I'm thinking about this more and wondering if what we should really do is make the indicators inline-block elements, considering their typical placement, and also give them a max-width of 100%. What you have now doesn't really address the issue at hand, which is the possibility of the indicator being wider than its container. For example (using a hardcoded 400px width on the `.gallery-example-live`):

Just adding `max-width: 100%;` solves the issue. I'm loathe to refactor the whole thing to be an `inline-block` element the layout is much cleaner and easier with flex box.

https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/3b9a69d92a6a8704a2ecdcfa56c3cc12ce1e40a0

<img width="306" alt="Screen Shot 2020-12-01 at 11 04 13 AM" src="https://user-images.githubusercontent.com/1991443/100765421-7d60c580-33c5-11eb-9ef7-da94509eb6a4.png">

Reply:
> I'm loathe to refactor the whole thing to be an inline-block element the layout is much cleaner and easier with flex box.

I guess I meant `inline-flex`.  Externally an inline element was my point, I wasn't suggesting changing the way the internals lay out.  They're meant to commonly be used in headers where they are supposed to lay out adjacent to the main header text.  Though I'll admit, the vertical alignment that's intended there isn't clear to me from the zeplin mockups.

Reply:
> > I'm loathe to refactor the whole thing to be an inline-block element the layout is much cleaner and easier with flex box.
> 
> I guess I meant `inline-flex`. Externally an inline element was my point, I wasn't suggesting changing the way the internals lay out. They're meant to commonly be used in headers where they are supposed to lay out adjacent to the main header text. Though I'll admit, the vertical alignment that's intended there isn't clear to me from the zeplin mockups.

I wish that I had remembered `inline-flex` existed: https://github.com/sonatype/sonatype-react-shared-components/pull/229/commits/7aca8373cc7d26a3c5ce19e22ee959749e1da968